### ISSUE TITLE
Buffs in buffs

### ISSUE THREAD

**Issue by Anthior** [Rughalt/D35E#304]
**The idea is to have buffs in buffs.** 
For example, the shifter race can activate their race ability (similar to the barbarian's rage) to gain a selected bonus between +2DEX, +2STR or +2CON. In addition of this single bonus, they also activate a "Shifter Mark" (which is something they choose at their character creation between something like 10 different available Marks). This Mark also give an additional bonus when activated, the said bonus being different depending on the Mark.

So, every shifter has access to the primary bonus (DEX, CON or STR) but they can just have 1 "Shifter Mark" at the start. But when they level up, they can choose a new Feat which give them another Mark.

**In conclusion**, at level 1, a shifter can activate his ability to get a bonus in STR, DEX or CON and the additional bonus linked to his Shifter Mark.
At level 9, a shifter can activate his ability to get a bonus in STR, DEX or CON and the several additional bonuses linked to all the Marks he has.

I really don't know if this is feasible tho, but just adding Buffs in Buffs will maybe be enough. 

Thank you again for all our work.
